### PR TITLE
8325073: javadoc warnings: missing @param tags and other issues

### PR DESCRIPTION
**Pull request by andy-goryachev-oracle**
This change brings the number of javadoc warnings back to 91 (to be fixed in [JDK-8270996](https://bugs.openjdk.org/browse/JDK-8270996))

- adds missing information in `@param` tags
 - adds `@SuppressWarnings("doclint:missing")` to `Skinnable` to silence the warning due to [JDK-8325071](https://bugs.openjdk.org/browse/JDK-8325071)
 - fixed an empty `<p>` in `Subscription`
 - cleaned up unnecessary `@throws` in Filtered/SortedList

This PR does not need a CSR: most of the changes are clarifications or documenting of missing parameters.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (2 reviews required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer), 1 [Author](https://openjdk.org/bylaws#author))

### Issue
 * [JDK-8325073](https://bugs.openjdk.org/browse/JDK-8325073): javadoc warnings: missing @param tags and other issues (**Bug** - P4)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Ambarish Rapte](https://openjdk.org/census#arapte) (@arapte - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1384/head:pull/1384` \
`$ git checkout pull/1384`

Update a local copy of the PR: \
`$ git checkout pull/1384` \
`$ git pull https://git.openjdk.org/jfx.git pull/1384/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1384`

View PR using the GUI difftool: \
`$ git pr show -t 1384`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1384.diff">https://git.openjdk.org/jfx/pull/1384.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1384#issuecomment-1970035990)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
> Q: Does this PR need a CSR?

Not if all of the changes are just clarification or documenting missing parameters.

**Comment by andy-goryachev-oracle** [comment]
@arapte @prsadhuk or @kevinrushforth could you please take a look?

**Review by kevinrushforth — COMMENTED** [review]
I did a quick pass, and confirm that there is nothing here that needs a CSR. There are enough changes that a second pair of eyes might be helpful, but not required.

I left a few inline comments. I might have more when I do a second pass.

File: modules/javafx.base/src/main/java/javafx/collections/transformation/FilteredList.java
Comment (andy-goryachev-oracle) [reply modules/javafx.base/src/main/java/javafx/collections/transformation/FilteredList.java:164]: yes, generated
```
FilteredList.java:164: warning: overridden methods do not document exception type java.lang.IndexOutOfBoundsException (module java.base package java.lang class IndexOutOfBoundsException)
     * @throws IndexOutOfBoundsException {@inheritDoc}
```

**Review by arapte — APPROVED** [review]
Looks good. Warnings reduced from 186 to 91.
One minor comment that at several instances there is no empty line between the description and `@param`. Shall re-approve if changed.

File: modules/javafx.controls/src/main/java/javafx/scene/control/ResizeFeaturesBase.java
Comment (kevinrushforth) [inline modules/javafx.controls/src/main/java/javafx/scene/control/ResizeFeaturesBase.java:34]: Minor: This class isn't limited to TableView, so I recommend changing it to something like "the control's items list"

File: modules/javafx.controls/src/main/java/javafx/scene/control/skin/ComboBoxListViewSkin.java
Comment (kevinrushforth) [inline modules/javafx.controls/src/main/java/javafx/scene/control/skin/ComboBoxListViewSkin.java:66]: Minor: should this be just "ComboBox" rather than "ComboBox-like"?

**Review by kevinrushforth — APPROVED** [review]
Looks good with two minor comments. I'll reapprove if you make the changes.

**Review by kevinrushforth — APPROVED** [review]

**Review by arapte — APPROVED** [review]

**Comment by andy-goryachev-oracle** [comment]
This PR is labeled READY but no integration email.  Is Scara down?

**Pull request closed without merging**
